### DISCUSSION TITLE
Generate Report for action without PR

### DISCUSSION THREAD

**Discussion by bslabiak** [andrcuns/allure-report-publisher#424] [General]
Hello,

Is this possible to attach test report for github action which is not triggered by other Pull Request? I have trouble to generate it because last 2 steps in the end are not starting:

"Adding reports urls
[✔] updating ... done"

If I am using same config but test are part of other PR everything is generated properly.

**Comment by andrcuns** [comment]
Sorry, I have never used `Discussions` so I missed this one.

You might want to check https://github.com/andrcuns/allure-publish-action for using this with GitHub actions.
